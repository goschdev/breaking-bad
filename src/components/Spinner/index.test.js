import React from "react"
import renderer from "react-test-renderer"

import { Spinner } from "."

describe("[Component] Spinner", () => {
  it("Should renderer", () => {
    const tree = renderer.create(<Spinner />).toJSON()

    expect(tree).toMatchSnapshot()
  })
})
