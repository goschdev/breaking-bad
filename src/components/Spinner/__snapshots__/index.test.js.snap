// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[Component] Spinner Should renderer 1`] = `
<div
  aria-hidden={true}
  className="sc-AxjAm jkdfwr"
>
  <img
    aria-hidden={true}
    className="sc-AxirZ sxpLU"
    src={Object {}}
  />
</div>
`;
